frontend/src/sagas/event.js

```diff
@@ -102,14 +102,9 @@ function connectTo(url) {
     return new Promise(resolver.bind(socket))
 }
 
-function* events() {
-    const protocol = location.protocol === "https:" ? "wss:" : "ws:";
-    const url = `${protocol}//${location.host}/ws`;
-
+function* events(socket) {
     while (true) {
         try {
-            console.log(`Connecting to backend: ${url}`)
-            const socket = yield call(connectTo, url)
```

Comment:
Well, at this point the reloading is more important. We need to figure out how to deal with unavailable backend both during startup and in the middle of using the app. We'll figure it out later.